src/execution/mod.rs

```diff
@@ -42,25 +42,33 @@ pub(crate) fn create_instance(limit: u64, module: &impl MassaModule) -> Result<I
     // Canonicalize NaN.
     compiler_config.canonicalize_nans(true);
 
-    // Turning-off in wasmer feature flags:
-    let mut features = Features::new();
-
-    // Disable threads.
-    features.threads(false);
-
-    // Turn-off experimental SIMD feature.
-    features.simd(false);
-
-    // Turn-off multivalue, because it is not supported for Singlepass(and it's true by default).
-    features.multi_value(false);
+    // Default: Turning-off all wasmer feature flags
+    // Exception(s):
+    // * bulk_memory:
+    //   * https://docs.rs/wasmer/latest/wasmer/struct.Features.html: now fully standardized - wasm 2.0
+    //   * See also: https://github.com/paritytech/substrate/issues/12216
+    const FEATURES: Features = Features {
+        threads: false, // disable threads
+        reference_types: false,
+        simd: false, // turn off experimental SIMD feature
+        bulk_memory: true,
```

Comment:
add a comment explaining why it is needed

Reply:
done